docs/source/topics/processes/usage.rst

```diff
@@ -350,6 +350,29 @@ The function will submit the calculation to the daemon and immediately return co
 .. warning::
     For a process to be submittable, the class or function needs to be importable in the daemon environment by a) giving it an :ref:`associated entry point<how-to:plugin-codes:entry-points>` or b) :ref:`including its module path<how-to:faq:process-not-importable-daemon>` in the ``PYTHONPATH`` that the daemon workers will have.
 
+.. tip::
+    Use ``wait=True`` when calling ``submit`` to wait for the process to complete before returning the node.
+    This can be useful for tutorials and demos in interactive notebooks where the user should not continue before the process is done.
+    One could of course also use ``run`` (see below), but then the process would be lost if the interpreter gets accidentally shut down.
+    By using ``submit``, the process is run by the daemon which takes care of saving checkpoints so it can always be restarted in case of problems.
+    If you need to launch multiple processes in parallel and want to wait for all of them to be finished, simply use ``submit`` with the default ``wait=False`` and collect the returned nodes in a list.
+    You can then pass them to :func:`aiida.engine.launch.await_processes` which will return once all processes have terminated::
```

Comment:
```suggestion
    You can then pass them to :func:`aiida.engine.launch.await_processes` which will return once all processes have terminated:
```

I think the extra colon is still causing trouble here.

![Screenshot 2023-10-27 at 17 35 40](https://github.com/aiidateam/aiida-core/assets/25607960/90dfce19-f11d-4955-ae34-a2f2c620b18d)

Reply:
Thanks for catching that. I tested the docs locally and it passed, but didn't bother to look at the actual output. Funnily enough, with just a single colon VSCode actually has incorrect syntax highlighting and lights up the whole phrase starting from `:func:` onwards. This was fixed by the double colon, but clearly that is not what restructured text wants.

I also noted that I forgot to update `sleep_interval` to `wait_interval` which I corrected now